portfolio/src/main/webapp/navbar.html

```diff
@@ -0,0 +1,6 @@
+<div class="topnav">
```

Comment:
Are you currently using this file somewhere? Or is this a placeholder you're adding before you make other changes?

Reply:
I'm hoping to write some CSS next, so I wanted some way to be able to reference it.